src/Wellcome.Dds/Wellcome.Dds.Common/AccessCondition.cs

```diff
@@ -18,17 +18,16 @@ public static class AccessCondition
         // Added for IIIF experiment
         public const string Degraded = "Degraded";
 
-        private static readonly List<ComparableAccessCondition> SecurityOrder
-            = new List<ComparableAccessCondition>
-                  {
-                      new ComparableAccessCondition(Open, 0),
-                      new ComparableAccessCondition(Degraded, 1),
-                      new ComparableAccessCondition(RequiresRegistration, 2),
-                      new ComparableAccessCondition(OpenWithAdvisory, 2),
-                      new ComparableAccessCondition(ClinicalImages, 3),
-                      new ComparableAccessCondition(RestrictedFiles, 4),
-                      new ComparableAccessCondition(Closed, 5)
-                  };
+        private static readonly List<ComparableAccessCondition> SecurityOrder = new()
```

Comment:
This is the best example of target-typed new I've seen in the wild, so much easier to read.